rust/lance-index/src/vector/ivf/transform.rs

```diff
@@ -106,6 +113,80 @@ impl<T: ArrowFloatType + L2 + Dot> IvfTransformer<T> {
 
         UInt32Array::from_iter(result.iter().flatten().copied())
     }
+
+    /// Compute the partition for each row in the input Matrix.
```

Comment:
do we use this in search? or just in build time.

Reply:
only in build time